packages/flutter_tools/lib/src/web/file_generators/flutter_js.dart

```diff
@@ -56,12 +56,53 @@ _flutter.loader = null;
     });
   }
 
+  /**
+   * Handles the creation of a TrustedTypes `policy` that validates URLs based
+   * on an (optional) incoming array of RegExes.
+   */
+  class FlutterTrustedTypesPolicy {
+    /**
+     * Constructs the policy.
+     * @param {[RegExp]} validPatterns the patterns to test URLs
+     * @param {String} policyName the policy name (optional)
+     */
+    constructor(validPatterns, policyName = "flutter-js") {
```

Comment:
Are default parameter values supported everywhere we want to run?

Reply:
Yes, I think this is Safari 10+, (but I'll deploy this somewhere online to give it a shot with real browsers.)

https://caniuse.com/mdn-javascript_functions_default_parameters

Reply:
I take this back, this hasn't been supported in Android until Chrome 105 !!!

Reply:
Deployed test app here: https://dit-tests.web.app, I took it for a spin with a bunch of browsers, simulators and emulators and this seems to work all right.